src/common/translation/i18n/fi.json

```diff
@@ -7,6 +7,10 @@
   },
   "appName": "Kulttuurin kummilapset",
   "authentication": {
+    "deviceTimeError": {
+      "message": "Usko tai et usko mutta et voi kirjaudua sisään koska laitteesi kello on yli 5 minuuttia väärässä. Säätää kelloa ja kokeile uudestaan. ",
+      "shortMessage": "Log in error. Check your device time and try again."
```

Comment:
```suggestion
      "shortMessage": "Virhe kirjautuessa. Tarkista laitteesi kello ja kokeile uudestaan."
```

Reply:
Thanks! I've updated the translation sheet, adding a new commit with this fix.